JuliaGPU/session2/Videh Raj Nema/a2q1.jl

```diff
@@ -0,0 +1,40 @@
+function partition(arr,start,stop)
+                  pivot=arr[stop]
+                  ind=start #start=1
+                  for i=start:(stop-1)
+                      if arr[i]<=pivot
+                          arr[i],arr[ind]=arr[ind],arr[i]
+                          ind=ind+1
+                      end
+                  end
+                  arr[ind],arr[stop]=arr[stop],arr[ind]
+                  #println(arr,"|",ind,"|",pivot) 
+                  return ind
+              end
+
+function quicksort(arr,start,stop)
```

Comment:
By convention in Julia, a mutating function (a function which modifies its parameters) is terminated with a '!'. So rename quicksort to quicksort! .